classes/Cart.php

```diff
@@ -3999,6 +3999,7 @@ public static function lastNoneOrderedCart($id_customer)
                 WHERE NOT EXISTS (SELECT 1 FROM ' . _DB_PREFIX_ . 'orders o WHERE o.`id_cart` = c.`id_cart`
                                     AND o.`id_customer` = ' . (int) $id_customer . ')
                 AND c.`id_customer` = ' . (int) $id_customer . '
+                AND c.`id_guest` <> 0
```

Comment:
Can we don't mix between `!=` and `<>`. We often use `!=`

Reply:
Applied and commit ammended